src/pages/wallets/index.tsx

```diff
@@ -117,7 +117,12 @@ const ChecklistItem = (props: HorizontalCardProps) => (
   <HorizontalCard
     border={0}
     display="flex"
-    alignItems="flex-start"
+    sx={{
+      "& > span": {
+        marginTop: "6.20px !important",
```

Comment:
Thanks @MahendraBishnoi29! Appreciate the attempt at precision 😆 but don't think we want to work with sub-pixel values here... 

I was going to suggest perhaps updating HorizontalCard to use a Grid so we could center align the first row with emoji and title, but this component is used in other places where this would not be appropriately. This component is designed to primarily adjust the emoji size, for example:

<img width="663" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/a685afe5-e992-4592-9454-b08f67fe44c3">

I think this particular usage with the green check looks strange because the emoji (16px) is smaller than the text next to it (18px).

A simply solution here could be to just increase the size of the check emoji to about 1.5rem by adding `emojiSize={1.5}` to the `ChecklistItem` declaration (as a prop to `HorizontalCard`) inside `pages/wallets/index.tsx`: 

<img width="707" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/8cc74a20-9054-4bb9-8eb3-81b80d58afa2">

@nloureiro or @konopkja Thoughts on this?

Reply:
I've made your suggested changes @wackerow @minimalsm